src/explore-education-statistics-admin/src/pages/admin-dashboard/components/MethodologySummary.tsx

```diff
@@ -43,6 +43,9 @@ const MethodologySummary = ({
     amendment: boolean;
   }>();
   const [dropMethodologyId, setDropMethodologyId] = useState<string>();
+  const [removeExternalMethodology, setRemoveExternalMethodology] = useState<
+    boolean
+  >(false);
```

Comment:
A better variable name would be `removingExternalMethodology` as it has a more stateful meaning. `removeExternalMethodology` sounds more like a function or the methodology that is being removed (when it's actually just a boolean).